app/admin/dashboard.rb

```diff
@@ -16,10 +14,20 @@
       end
 
       column do
-        panel "Info" do
-          para "Welcome to ActiveAdmin."
+        panel 'Info' do
+          para 'Welcome to ActiveAdmin.'
         end
       end
     end
+
+    section 'This month' do
+      table_for Event.where('datetime > ? and datetime < ?',
```

Comment:
DateTime.now.beginning_of_day + 1.days -> for future this exp can be replaced with 1.day.from_now